examples/retry_command.ts

```diff
@@ -0,0 +1,34 @@
+import { Command, CommandStep, Pipeline } from '../src';
+import { YamlSerializer } from '../src/serializers/yaml';
+
+class RetryCommand extends Command {
+    constructor(public retries: number, public c: Command) {
+        super(c.toString());
```

Comment:
```suggestion
        super(c.command, c.timeout);
```

you need to pass on the timeout, otherwise it gets lost from the original command - I think this should be:

```suggestion
        super(c.command, c.timeout * (this.retries + 1));
```

however